src/pages/my_page/MyPagePage.tsx

```diff
@@ -89,6 +89,12 @@ const MyPagePage = () => {
             to={UrlPathTemplate.MyPageMyProfile}>
             {t('my_page.my_profile.user_profile')}
           </LinkButton>
+          <LinkButton
+            data-testid={dataTestId.myPage.myPublicationsLink}
```

Comment:
siden det ble nytt navk på knappe
```suggestion
            data-testid={dataTestId.myPage.myResultsLink}
```